Rubrik/Public/Set-RubrikSettings.ps1

```diff
@@ -0,0 +1,91 @@
+﻿#Requires -Version 3
+function Set-RubrikSettings
+{
+  <#  
+      .SYNOPSIS
+      Connects to Rubrik and sets Rubrik cluster settings
+            
+      .DESCRIPTION
+      The Get-RubrikSettings cmdlet will set the cluster settings on the system. This does require authentication.
+            
+      .NOTES
+      Adapted by Adam Shuttleworth from scripts by Chris Wahl for community usage
+            
+      .LINK
+      https://github.com/rubrikinc/PowerShell-Module
+            
+      .EXAMPLE
+      Set-RubrikSettings -Server "192.168.1.100" -clusterName "test-rubrik-cluster" -timeZone "America/Los Angeles" -clusterLocation "LA Office"
+      This will set the designated cluster settings on the Rubrik cluster reachable at the address 192.168.1.100
+  #>
+
+  [CmdletBinding(SupportsShouldProcess = $true,ConfirmImpact = 'High')]
+  Param(
+    # Rubrik server IP or FQDN
+    [Parameter(Mandatory = $true)]    
+    [String]$Server,
+    # ID of the Rubrik cluster or me for self
+    [String]$id = 'me',
+    # API version
+    [ValidateNotNullorEmpty()]
+    [String]$api = 'v1',
```

Comment:
The $Server and $api vars needs to point to the global connection settings, otherwise it won't work on first run and you'll be stuck on API v1 in the future :)

```
    # Rubrik server IP or FQDN
    [String]$Server = $global:RubrikConnection.server,
    # API version
    [String]$api = $global:RubrikConnection.api
```

Reply:
This should be fixed throughout all the new functions.